PoW-vs-Pos/en/en_transcription.md

```diff
@@ -0,0 +1,102 @@
+
+(21:57) cpu cores 128 gigabytes of ram and 300 megabit a second upload speed that setup especially the upload speed part basically means you need to be a data center operator to run a solana validator secondly even those data center level validators have to rely on archivers if they want to go back through the full history of the blockchain because over time the amount of stored information becomes massive thirdly solana uses manual slashing in other words it's a blockchain that requires human decisions to determine
+
+(22:31) consensus if there are significant attacks on the network so solana is not really even an automated blockchain it's a step back from both proof-of-work bitcoin and proof-of-work ethereum in terms of automation in exchange for more throughput and low fees the consensus mechanism is more manual human and political smart contract platforms tend to move towards greater and greater centralization because the more centralized they are the more efficient they are and users want efficiency i've not gone to the step of saying that
+
+(23:05) people should have ethereum and it's because of that centralization problem that essentially it's this environment where you're changing the engine of a car while you're driving it so they're gonna change from proof of work to proof of stake it's been it's been delayed for years and because it's a hard problem it's a very hard problem they've also they keep changing how they're going to scale it there's been inventions over the past four or five years that that
+
+(23:28) have changed some of their their base case for how they're going to scale their protocol and so overall my my issue with some of the smart contract platforms is that they seem like they're going to tend towards centralization and and you see that with the solana versus ethereum comparison where a more centralized blockchain comes along that's more efficient because it's more centralized and then people start kind of flocking to it the excitement goes there the the d5 grows a little bit quicker there than on ethereum because
+
+(24:00) there's lower fees and again they don't fully understand the trade-offs but as long as it's working they don't really mind too much as a result smart contract platforms that have higher throughput and a massive support each have a decent shot of taking market share the field keeps diluting itself with more efficient and more centralized networks however ethereum also partially goes down from time to time with unintended chain splits and it's possible that if it changes over to proof of stake it could
```

Comment:
Distinguere le frasi non del narratore `i've not gone to the step of saying that people should have ethereum and [...] there's lower fees and again they don't fully understand the trade-offs but as long as it's working they don't really mind too much`